app/mailers/contact_form_mailer.rb

```diff
@@ -4,6 +4,6 @@ def contact_form_mail(sender_email, recipient_email, name, subject, message)
     @name = name
     @recipient_email = recipient_email
   
-    mail(to: [sender_email, recipient_email], subject: subject, from: 'contactform@bedlamtheatre.co.uk', reply_to: sender_email)
+    mail(to: [sender_email, recipient_email], subject: subject, reply_to: sender_email)
```

Comment:
It doesn't really matter I think since it won't ever be replied to (there's a specific reply_to which is the user-submitted email) so it might as well be consistent I think?